From 70ff12a694a85184775740fcb501342c75fd551b Mon Sep 17 00:00:00 2001
From: Andriy Grytsenko <andrej@rep.kiev.ua>
Date: Tue, 20 Nov 2012 23:53:16 +0200
Subject: [PATCH 18/22] Change FM_CONFIG_DEFAULT_NO_USB_TRASH to more expecting
 TRUE.

The most expected for users used to Windows behavior for that option
is to have it TRUE, and FALSE creates behavior unexpected by most of
the users and, moreover, even when user plug off flash drive an then
plug in it, the trash may be not always recognized by GVFS therefore
it's not trivial to get files on flash drive be REALLY deleted. And
users blame pcmanfm, of course.
The commit changes default. It will not affect users who already use
libfm 1.1.0 and have own setting for it but only new users.
---
 src/base/fm-config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/base/fm-config.h b/src/base/fm-config.h
index 313db7c..cd41399 100644
--- a/src/base/fm-config.h
+++ b/src/base/fm-config.h
@@ -43,7 +43,7 @@ typedef struct _FmConfigClass       FmConfigClass;
 #define     FM_CONFIG_DEFAULT_SINGLE_CLICK      FALSE
 #define     FM_CONFIG_DEFAULT_USE_TRASH         TRUE
 #define     FM_CONFIG_DEFAULT_CONFIRM_DEL       TRUE
-#define     FM_CONFIG_DEFAULT_NO_USB_TRASH      FALSE
+#define     FM_CONFIG_DEFAULT_NO_USB_TRASH      TRUE
 
 #define     FM_CONFIG_DEFAULT_BIG_ICON_SIZE     48
 #define     FM_CONFIG_DEFAULT_SMALL_ICON_SIZE   16
-- 
1.8.0.1

